scripts/jobs/cron_tasks.inc.http.10.apache.php

```diff
@@ -882,108 +903,105 @@ protected function getVhostContent($domain, $ssl_vhost = false)
 			}
 			$ipportlist .= $ipport;
 		}
+
```

Comment:
what are the massive changes below? They do not seem to have anything to do with IP/Proxy stuff...irritating

Reply:
I had to refactor the code a bit because the creating of the vHost code was mixing with setting of properties. I had the issue, that there were vHost container created with an empty $ipportlist, which is now captured later.